src/mongo/delegates/Updates.class.php

```diff
@@ -585,13 +592,14 @@ protected function applyChangeSet(\Tripod\ChangeSet $cs, $originalCBDs, $context
                 $command = array(
                     "findAndModify" => $this->getCollection()->getName(),
                     "query" => $update['criteria'],
-                    "update" => $update['change'],
+                    "update" => $update['changes'],
                     "upsert" => $upsert,
                     "new"=>true
                 );
 
                 try{
                     $result = $this->getDatabase()->command($command);
```

Comment:
They effectively do the same thing, MongoCollection::findAndModify() just seems a little nicer, idiomatically.